content/docs/tools/teddyCloud/_index.md

```diff
@@ -12,11 +12,22 @@ to host your own figurine audio files on e.g. your [NAS or any other server](sup
 You can also use teddyCloud on the command line to manipulate esp32 firmware dumps or encode Tonie Audio Files (TAFs). See ```toniecloud --help```.
 
 ## Docker hints
+
+### General
 The docker container automatically generates the server certificates on first run. You can extract the ```certs/server/ca.der``` for your box after that. 
 
 An example [docker-compose.yaml can be found within the docker subdir.](https://github.com/toniebox-reverse-engineering/teddycloud/blob/master/docker/docker-compose.yaml)
 Please beware that port 443 cannot be remapped and you cannot use a reverse proxy like nginx or traefik without passing through the TLS (complex, not recommended). The client certificate authentication needs to be done by teddyCloud. Also, there is no SNI. If you are using docker, you can use macvlan to give the teddyCloud container a dedicated IP address (recommended).
 
+### Windows Quick Start
++ Get https://www.docker.com/products/docker-desktop/
++ Get the files which you can find [here](https://github.com/toniebox-reverse-engineering/teddycloud/blob/master/docker/windows)
```

Comment:
There is only one file. 
Beside that I would suggest using the original docker compose file and let the user edit it themselves. Otherwise, the composefile needs to be duplicated.